python/lsst/daf/butler/remote_butler/_remote_butler.py

```diff
@@ -287,9 +289,26 @@ def getURI(
         collections: Any = None,
         run: str | None = None,
         **kwargs: Any,
-    ) -> ResourcePath:
+    ) -> DatasetRefURIs:
         # Docstring inherited.
-        raise NotImplementedError()
+        if predict or run:
+            raise NotImplementedError("Predict mode is not supported by RemoteButler")
```

Comment:
We need to discuss whether we can make this work. Obviously a signed URL to a resource that doesn't exist is not all that helpful but people like to know what the filename might look like. We can tell them that in theory -- maybe for predict mode we return the unsigned `s3` URI? Can punt this to another ticket.